setup.cfg

```diff
@@ -1,9 +1,6 @@
 [aliases]
 test=pytest
 
-[tool:pytest]
-addopts = -v -rsXx
-
```

Comment:
These were being ignored due to the existence of pytest.ini in `tests/`.

Reply:
👍